dependencies.json

```diff
@@ -144,6 +161,6 @@
   {
     "Kind": "GitSubmodule",
     "Name": "External/Xamarin.PropertyEditing",
-    "Version": "953d37877f78fdf1fa99a33e189aa99a6b5f4655"
+    "Version": "1ad39236886f30dded16811e588ae4accddc4a19"
```

Comment:
Is this intentional?

Reply:
yeah it is correct

Reply:
Yes, this was missing from @lewing's bump. It reflects the ref on master.